module/users/validator/v1.js

```diff
@@ -12,6 +12,10 @@ module.exports = (req) => {
         targetedEntity : function () {
             req.checkParams("_id").exists().withMessage("required solution id");
             req.checkBody("role").exists().withMessage("required user role");
+        },
+        getAllStats:function(){
+            req.checkQuery('type').optional().isIn(['observation', 'project', 'survey', 'program']) // If present, it must be one of these values
```

Comment:
what if its not passed then @borkarsaish65

Reply:
It will give the big numbers. if type is sent then detailed information is fetched.